src/guides/resolve-zil-domain.md

```diff
@@ -0,0 +1,418 @@
+---
+description: Simplest step-by-step guide on how to resolve .zil domain
+---
+
+# Resolve a .zil domain
+
+## Where can I get .zil domain?
+The most simpliest way is to go to [our homepage](https://www.unstoppabledomains.com) and buy one for yourself
+## Plan
+
+![Demo](../.gitbook/assets/demozil.gif)
+
+In order to resolve a .zil domain, we will need to satisfy three steps below
+* [Get a namehash of a domain](resolve-zil-domain.md#taking-a-namehash)
+* [Get resolver contract address from a domain](resolve-zil-domain.md#getting-resolver-address)
+* [Query resolver contract to fetch the records](resolve-zil-domain.md#fetching-the-domain-records)
+* [Displaying results](resolve-zil-domain.md#displaying-result)
+* [Error handling](resolve-zil-domain.md#error-handling)
+
+## Initialize the project folder
+
+We are going to build the project using [parcel.js](https://parceljs.org/) as it allows us to include cryptographic library [hash.js](https://www.npmjs.com/package/hash.js/v/1.1.7) for encoding the domain in the future and typescript supports from the start. For now, let's open a terminal and initialize the project.
+
+```text
+npm install -g add parcel-bundler
+mkdir unstoppable-zil-resolution
+cd unstoppable-zil-resolution
+npm init
+npm install hash.js
+touch index.ts index.html
+
+
+.
+├── README.md
+├── index.html
+├── index.ts
+├── node_modules
+├── package.json
+
+```
+
+Now open **package.json** and configure run scripts for us
+
+```javascript
+  "scripts": {
+    "dev": "parcel index.html",
+    "build": "parcel build index.html"
+  }
+```
+
+As our next step, let's create a blank HTML page. Create **index.html** and use the following code.
+
+```markup
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Basic .zil integration</title>
+    </head>
+    <body>
+      <div id="main" style="
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+      ">
+        <input id="input" />
+        <button id="button">Resolve</button>
+        
+        <div id="records" style="display: flex; flex-direction: column;">
+
+        </div>
+      </div>
+      <script src="./index.ts"></script>
+  </body>
+</html>
+```
+
+{% hint style="info" %}
+Take notice of this line `code <script src="./index.ts"></script>.` Parcel package will automatically convert this line to the js bundle it generates from our code.
+{% endhint %}
+
+### **Create** ~~**index.js**~~  **index.ts**
```

Comment:
@JohnnyJumper 

This is how it looks right now. 
<img width="261" alt="image" src="https://user-images.githubusercontent.com/3999180/106001597-32abb300-60c1-11eb-9070-5a1589ae500d.png">

Can you edit markdown in the editor instead of gitbook? I believe vscode has great options for editing, linting and preview markdown: https://code.visualstudio.com/docs/languages/markdown